src/db.ts

```diff
@@ -14,7 +13,7 @@ export type Transaction = Knex.Transaction;
 
 export class DB {
 	private databasePath: string;
-	private knex: Knex;
+	public knex: Knex;
```

Comment:
NIT: Why make it public ?

Reply:
I also wonder about this @CameronDiver ?

Reply:
Ah, leftover debugging. Good catch.